src/JustSaying/Fluent/TopicSubscriptionBuilder`1.cs

```diff
@@ -17,10 +19,13 @@ public sealed class TopicSubscriptionBuilder<T> : ISubscriptionBuilder<T>
         /// <summary>
         /// Initializes a new instance of the <see cref="TopicSubscriptionBuilder{T}"/> class.
         /// </summary>
-        internal TopicSubscriptionBuilder()
+        internal TopicSubscriptionBuilder(SubscriptionsBuilder parent)
```

Comment:
Docs.

Reply:
Removed this parameter as it wasn't used 👍